### PR TITLE
Backend

### PR DESCRIPTION
**Pull request by gsrai**
now works with firebase! no admin login authentication yet.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "pool-request",
-  "version": "0.0.0",
+  "version": "0.2.1",
```
Comment (lukehowkins) [inline package.json]: Didn't realise we were versioning.

File: tests/unit/routes/admin-test.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('route:admin', 'Unit | Route | admin', {
+  // Specify the other units that are required for this test.
+  // needs: ['controller:foo']
+});
+
+test('it exists', function(assert) {
+  let route = this.subject();
```
Comment (lukehowkins) [inline tests/unit/routes/admin-test.js]: Not much point of these tests.

File: package.json (hunk 1 above)
Comment (gsrai) [inline package.json]: Got to play like we're in the big leagues, versioning makes our software legit.

File: app/mirage/config.js [hunk 3]
```diff
@@ -117,16 +117,16 @@ export default function() {
...
-    var params = JSON.parse(request.requestBody).data.attributes;
-    params.type = 'game';
-
-    if (!params['name-one'] || !params['name-one']) {
-      return new Mirage.Response(400, { a: 'header' }, { message: 'name cannot be blank' });
-    } else {
-      return {data: db.games.insert(params) };
-    }
-  });
+  // this.post('/games', function(db, request) {
```
Comment (lukehowkins) [inline app/mirage/config.js]: Why is the post the only one you need to comment out?

Comment (gsrai) [inline app/mirage/config.js]: ye i need to uncomment it.

File: tests/unit/routes/admin-test.js (hunk 2 above)
Comment (gsrai) [inline tests/unit/routes/admin-test.js]: ill write them in another pr, MVP BRO

File: package.json (hunk 1 above)
Comment (lukehowkins) [inline package.json]: Just making up the number are you or is there some logic to it?

Comment (gsrai) [inline package.json]: well i added two changes and one tiny patch, i was gonna update major version to 1 but that would denote that pool request is out of alpha and beta and prod ready. What do you think?

Comment (lukehowkins) [inline package.json]: Definitely not a major version update. But I would have assumed you would have gone off the pull requests

Comment (gsrai) [inline package.json]: like nwtp? ill change it, to 0.6.0

**Pull request merged**